workshops/test/test_person.py

```diff
@@ -119,7 +122,7 @@ def _check_person(self, doc, person):
                     .format(key, value, type(value), node.text)
 
     def _get_field(self, doc, key):
-        '''Get field from person display.'''
+        """ Get field from person display. """
```

Comment:
The rule of thumb is to fix not-PEP8-compliant code when you edit it, and not just fix it because you want to.

And I'm very sorry because from time to time I do not follow this rule :-)

So in this case I'd just leave the additional commit as it is.

Question though: I've seen you use `""" spaces right next to the triple quotes. """`. Is it something dictated by PEP8 / PEP257?